src/py21cmfast/wrapper.py

```diff
@@ -2009,6 +2085,12 @@ def __init__(
         node_redshifts=None,
         global_xHI=None,
         global_brightness_temp=None,
+        analytic_z = None,
```

Comment:
can we put  all of these into a dictionary called something like `photon_nonconservation_data`? Also name the variables with `snake_case`, not `CamelCase`.

Reply:
Concatenated this into a single dictionary. Note, this now adds an extra returned argument to the run_coeval function which will need to be appropriately added to the examples etc.